### ISSUE TITLE
normalization of predicted and ground truth

### ISSUE THREAD

**Issue by bachw433** [JunjH/Revisiting_Single_Depth_Estimation#6]
hi,
thanks for your sharing. it's really amazing.
but I'm a little bit confused about the depth normalization 

I noticed that in your code, all the traing image are normalized into [0, 1] before training,
what about the depth? do you also scaled ground truth depth into [0, 1] as well?
It seems reasonable at first beginning.
But if you apply log on the range[0, 1], it will become negative. does it make sence?

**Comment by JunjH** [comment]
We use real depth value (0~10 meters)  for training.
For training the network, the important thing is the derivative of loss function.
